atlas/customization/builds.mdx

```diff
@@ -0,0 +1,73 @@
+---
+slug: /atlas/customization/builds
+title: Configuring Atlas Builds
+description: Learn how to configure builds on the Atlas platform
+---
+
+Here we describe ways of configuring the Atlas build system for your needs. You can configure the Node.js environment, how dependencies are installed and what command is used to build your app.
+
+## Setting a custom version of Node.js, npm and yarn
+
+### Node.js
+
+By default, Atlas uses the latest LTS Node.js version which is v16. The timeline of LTS releases is tracked [here.](https://nodejs.org/en/about/releases/) Currently we maintain support for Node.js v12, v14, v16.
+
+To set a custom version of Node.js, use the [engines](https://docs.npmjs.com/cli/v8/configuring-npm/package-json#engines) field in the `package.json` file. For example here is how to set it to use Node.js 14.
+```json
+"engines": {
+  "node": "v14"
+},
+```
+
+To see what version of Node.js you are using on your local machine use the following command:
+```bash
+$ node --version
+v16.15.0
+```
+
+### npm
+
+The default version of npm used depends on the version of Node.js you have selected. You can use [this page](https://nodejs.org/en/download/releases/) to search which version of npm comes with a particular version of Node.js. Alternatively to see what version you have installed locally use:
+```bash
+$ npm --version
+8.11.0
+```
+
+Atlas looks for your npm version in two places. The first is the `engines` field in the `package.json` file:
+```json
+"engines": {
+  "npm": "8.11.0"
+}
+```
+
+An alternative way to set npm is to use the `packageManager` field in the `package.json` file. Note the difference in format compared to the `engines` field above.
+```json
+"packageManager": "npm@8.11.0"
+```
+
+### yarn
+
+Atlas looks for a `yarn.lock` file to decide if your app is using yarn. If no `yarn.lock` file is present then it defaults to `npm`. 
+
+Currently only yarn classic (v1) is supported with the default version on Atlas being `v1.22.18`. Other versions of yarn classic can be selected by using the `packageManager` field in `package.json`
```

Comment:
I'd be in favour of not having the exact version of yarn we use in this doc, just because we're going to forget to keep it updated.